tutorials/math/normalDist.py

```diff
@@ -0,0 +1,96 @@
+## \file
+## \ingroup tutorial_math
+## \notebook
+## Tutorial illustrating the new statistical distributions functions (pdf, cdf and quantile)
+##
+## based on Anna Kreshuk's normalDist.C
+##
+## \macro_image
+## \macro_code
+##
+## \author Juan Fernando Jaramillo Botero
+
+from ROOT import TF1, TCanvas, TSystem, TAxis, TLegend
+from ROOT import kRed, kGreen, kBlue
+
+
```

Comment:
Hi @juanfe, thanks for addressing the comments! I have one final comment for what concerns style: when this is translated into a notebook, every group of consecutive code lines will be put into a separate cell. If before some group of code lines (e.g. where I just added this comment) you add one or more comment lines, those lines will be translated into a markdown cell of a notebook.
Therefore, feel free to add comments to explain what is being done in your code, and that will result in a better-looking notebook as well!